libmscore/cmd.cpp

```diff
@@ -1563,14 +1563,6 @@ void Score::upDown(bool up, UpDownMode mode)
                   }
 
             if ((oNote->pitch() != newPitch) || (oNote->tpc1() != newTpc1) || oNote->tpc2() != newTpc2) {
-                  // remove accidental if present to make sure
-                  // user added accidentals are removed here.
-                  auto l = oNote->linkList();
-                  for (ScoreElement* e : l) {
-                        Note* ln = toNote(e);
-                        if (ln->accidental())
-                              undo(new RemoveElement(ln->accidental()));
-                        }
```

Comment:
The other thing we could do is only execute this code if `mode != UpDownMode::OCTAVE`. Presumably this code was added for a reason. See https://github.com/musescore/MuseScore/commit/477aafd.

Reply:
Yes, right, I was being foolish.